earth_enterprise/src/fusion/autoingest/sysman/gefusion

```diff
@@ -117,8 +157,12 @@ case "$1" in
     stop)
     stop
     ;;
+    status)
+	status
```

Comment:
Minor: This is really the correct indention, but it would be better to match the other cases, so I'd suggest either updating the other cases to match this one or this one to match the others

Reply:
The readability boost is totally worth it, will do.